addons/website_slides/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -125,6 +125,22 @@
                                     <i class="fa fa-link mr-2"/><span t-esc="link.name"/>
                                 </span>
                             </li>
+                            <div class="o_wslides_js_course_join pl-0" t-if="slide.resource_ids">
+                                <t t-if="is_member">
```

Comment:
you can put the t-if inside the li t-foreach (at the beginning to be more clear with the t-elif that follows)
do the same in addons/website_slides/views/website_slides_templates_lesson.xml